### DISCUSSION TITLE
Adjust Video Settings and Volume?

### DISCUSSION THREAD

**Discussion by RandomMan719** [GerryDazoo/Slinger#233] [Q&A]
Gerry, will Slinger have any options to adjust video settings and volume like when you configure a slingbox?  You get Brightness, Contrast, Saturation, Hue, and Volume adjustments.  I would think those settings would be saved in the slingbox, but I have no idea.  Thanks again, for this project.

**Comment by GerryDazoo** [comment]
I think those options are on the player side not the slingbox. Check the docs for your favourite media player, that's where I would start looking. I.e.
https://www.vlchelp.com/increase-brightness-contrast-videos/

**Reply by RandomMan719** [reply]
I will do a test, real quick, by setting one up with low contrast and see if it is just the player.  Thanks.

**Reply by RandomMan719** [reply]
I instead adjusted the hue and saturation, with Slingplayer 2.0 setup assistant, picture settings.  I then fired up Slinger and connected to it and the picture kept the adjustment, so I believe those values are stored in the slingbox itself.  I tested this with a ProHD model, using the composite input (VideoSource=1) with a converterbox hooked up, not on the internal tuner.  I haven't tested any other models, but I assume they are all this way.  Here's a screenshot.
![Hue   Saturation Adjusted](https://user-images.githubusercontent.com/116921195/200086712-c1a56e47-1fc5-45b2-b7ba-711c5f940180.PNG)

**Reply by sp222222** [reply]
i’ll say this. in all my years doing setups the only thing i ever messed with was the volume slider. the
picture always looked the same for me on the device. you might want to check your video connection cables and use quality vid/aud cabling.

**Reply by RandomMan719** [reply]
The cables are fine, I did that as a test to show that the video and volume settings are stored in the slingbox.

**Reply by GerryDazoo** [reply]
You've convinced me. Too bad we don't know how to change it. Yet....

**Reply by RandomMan719** [reply]
Yeah, it's not a big deal.  I just wanted to know how it works, or if you knew how to change those settings.  Thanks.
